libs/net/models/Areas/Editor/Models/Content/WorkOrderModel.cs

```diff
@@ -70,6 +86,50 @@ public WorkOrderModel(Entities.WorkOrder entity)
         this.AssignedId = entity.AssignedId;
         this.Description = entity.Description;
         this.Note = entity.Note;
+        if (entity.Requestor != null)
+            this.Requestor = new UserModel(entity.Requestor);
+        if (entity.Assigned != null)
+            this.Assigned = new UserModel(entity.Assigned);
+        // if (entity.Content != null)
```

Comment:
Not sure why this is causing an error? Currently headline not being populated as I cannot get this to work